ui/admin/app/routes/scopes/scope/add-storage-policy/index.js

```diff
@@ -23,12 +23,20 @@ export default class ScopesScopeAddStoragePolicyIndexRoute extends Route {
   async afterModel() {
     const { id: scope_id } = this.modelFor('scopes.scope');
 
-    //fetch policies from current scope
-    const orgScopePolicies = await this.store.query('policy', {
-      scope_id,
-    });
-
-    this.policyList = orgScopePolicies;
+    if (scope_id === 'global') {
+      //Global scope should list both global and org scope policies
```

Comment:
```suggestion
      // Global scope should list both global and org scope policies
```

Can we add a space to initial comments?